test/goldens/end2end/fs_shim-0.7.1.json_report.md

```diff
@@ -82,10 +82,13 @@ ERROR: The default value of an optional parameter must be constant.
 To reproduce run `dart analyze lib/fs.dart`
 </details>
 
-## 20/30 Package is a good, trustworthy dependency
+## 10/30 Package is a good, trustworthy dependency
 
-### [*] 10/10 points: All of the package dependencies are supported in the latest version
+### [x] 0/10 points: All of the package dependencies are supported in the latest version
 
+* Could not run pub outdated: Exception: `pub get` exited 1. StdErr: The current Dart SDK version is {{sdk-version}}.
+
+Because fs_shim requires SDK version >=1.24.0 <2.0.0, version solving failed.
```

Comment:
Let's make this:
```suggestion
* Could not run pub outdated:
```
The current Dart SDK version is {{sdk-version}}.

Because fs_shim requires SDK version >=1.24.0 <2.0.0, version solving failed.
```
```

Reply:
Ah, sorry, I've missed this comment, I'll do a follow-up on that.